doc/api/buffer.md

```diff
@@ -3279,6 +3279,28 @@ While, the `Buffer` object is available as a global, there are additional
 `Buffer`-related APIs that are available only via the `buffer` module
 accessed using `require('buffer')`.
 
+### `buffer.atob(data)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `data` {any} The Base64-encoded input string.
+
+Decodes a string of Base64-encoded data into ASCII (Latin1).
+
+The `data` may be any JavaScript-value that can be coerced into a string.
+
+### `buffer.btoa(data)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `data` {any} An ASCII (Latin1) string.
+
+Encodes a string of ASCII characters into a Base64-encoded string.
+
+The `data` may be any JavaScript-value that can be coerced into a string.
```

Comment:
If we do implement this, we should note that a) these methods are only provided for browser compatibility and b) that **you should not use them** and why, and convert between Buffers and strings using `Buffer.from(str, 'base64')` and `buf.toString('base64')` instead.